av/container/core.pxd

```diff
@@ -4,6 +4,11 @@ from av.container.streams cimport StreamContainer
 from av.dictionary cimport _Dictionary
 from av.format cimport ContainerFormat
 from av.stream cimport Stream
+from posix.time cimport timeval
+
+ctypedef struct cb_info:
+    int timeout
```

Comment:
Maybe store the timeout directly in milliseconds and likewise the start time in milliseconds, that way we don't drag in `timeval` here.